includes/functions.php

```diff
@@ -998,6 +998,12 @@ function tsml_get_meeting($meeting_id=false) {
 	$meeting->post_title			= htmlentities($meeting->post_title, ENT_QUOTES);
 	$meeting->notes 				= esc_html($meeting->post_content);
 
+	// Remove TC when online only meeting has approximate address
```

Comment:
i'm not sure why this is needed -- shouldn't these meetings not be TC?

Reply:
No. An online meeting that has an approximate address doesn't need TC because we view it as either natively online or it is a meeting that because online and the in-person option went away *permanently*. We only view TC as necessary when we need to tell the user a specific address is not currently available to attend in person.

Unless we've really goofed, that should be consistent with the spec.

Reply:
Right that's my understanding too - so not sure why we're trying to filter it out on the meeting page here when it shouldn't be in the database at all.

Reply:
I believe this is because we're cleaning up an issue we introduced in which *all* online meetings initially got tagged as TC. Is that correct @brianw-area24?.

Reply:
well ok if that's an issue then let's add a database cleanup operation - not saying it needs to be done in this PR, we could create a future ticket like "clean up TC from approximate meetings and remove logic from tsml_get_meeting"

Reply:
My suggestion would be to remove lines 1001-1004 now, and leave taking TC out of the database for a future pull release. We should probably do an issue around this...

Reply:
Ok thanks for spotting that Brian. I've removed it now and created this issue: https://github.com/code4recovery/12-step-meeting-list/issues/493